lms/djangoapps/courseware/views.py

```diff
@@ -366,6 +376,22 @@ def _index_bulk_op(request, course_key, chapter, section, position):
             user.id, unicode(course.id))
         return redirect(reverse('dashboard'))
 
+    # see if course has entrance exam enabled then it should be passed by user
+    # before accessing any other chapter/section of course
+    # Note that if the entrance exam feature flag has been turned off (default)
+    # then this check will always pass
```

Comment:
Must: please reword these comments. The first one ("see if course...") is hard for me to read. The second one states that "this check will always pass" in the case when the feature flag is off, but since this conditional will fail in that case, and since people associate "conditional" with "check", I suggest rewording that to say that, in the case where the feature flag is off, we never do the following redirect.